src/JustSaying/Messaging/Channels/RoundRobinQueueMultiplexer.cs

```diff
@@ -128,9 +127,10 @@ private async Task RunImpl()
             }
         }
 
-        public async IAsyncEnumerable<IQueueMessageContext> Messages()
+        public async IAsyncEnumerable<IQueueMessageContext> GetMessagesAsync()
         {
-            if (!_started) throw new InvalidOperationException("");
+            if (!_started) throw new InvalidOperationException(
+                "Multiplexer must be started before listening to messages");
```

Comment:
I'm not a fan of eliding braces (but won't ask you to change them all) as I think it's less readable and creates more churn when code needs to change, but this one is confusing spreading over more than one line.

```suggestion
            if (!_started)
            {
                throw new InvalidOperationException("Multiplexer must be started before listening to messages");
            }
```

Reply:
No problem here, will add a rule to editorconfig to fix on format. I think in library scenarios its ok to enforce always using braces.